submissions/m-ruslan/memory-pair-game/app.js

```diff
@@ -0,0 +1,135 @@
+// main logic of game
+document.addEventListener("DOMContentLoaded", () => {
+  let cardID, cardValue, prevCardId, prevCardValue;
+  let qOfOpenCards = 0;
+
+  startNewGame();
+
+  document.querySelector("main").addEventListener("click", (event) => {
```

Comment:
[separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns)

too big function. what it do ? check cards, check win condition etc.
 So it better to have smaller, more abstract functions, than complex like this one